scenarios/topo/sliceMeterConfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "sliceName" : "MOBILE",
```

Comment:
I know the name mobile slice is there for historical reasons, but lit's becoming too generic. `P4-UPF-1` would be a more accurate name considering that there will be more than one P4-UPF slice, and we plan to have slices to isolate BESS-UPF traffic. Eventually, we should replace all existing references of "mobile slice" in other parts of the codebase.